README.md

```diff
@@ -114,6 +114,24 @@ Options:
   --help             Show help                                                                                 [boolean]
 ```
 
+### How to deploying a new DAO:
+1. Make sure you have Node.js and NPM installed and updated to a version later that 10.0.0 and 6.0.0 respectively. You can find instructions for installation [here](https://nodejs.org/en/download/), and then verify your version by opening your CMD/ Terminal and type: `node -v` and then `npm -v`.
+
+2. In your terminal, use this command to install the DAOstack migration CLI: `npm install --global @daostack/migration`.
```

Comment:
please add both CLI and JS script options.

Reply:
There's already a JS explanation in the beginning.

Reply:
The JS is not clear . Is that working with the new script?

Reply:
No reason it won't, why is it not clear?

Reply:
what is 
```
DAOstackMigration.Ganache.server(..);
DAOstackMigration.Ganache.provider(..);
```

Reply:
Why 'private' ?
`DAOstackMigration.migration('private');`

Reply:
We need a simple instruction to setup a custom dao . not the whole migration.
as below...
```
// migrate base contracts
const migrationBaseResult = await DAOstackMigration.migrateBase(options);
migrationBaseResult.base.GenesisProtocol // migrated genesis protocol address
// migrate an example DAO (requires an existing `output` file with a base migration)
const migrationDAOResult = await DAOstackMigration.migrateDAO(options);
migrationDAOResult.dao.Avatar // DAO avatar address
// migrate an demo test scenario (requires an existing `output` file with a base migration)
const migrationDemoResult = await DAOstackMigration.migrateDemoTest(options);
migrationDemoResult.test.Avatar // Test DAO avatar address
// migrate base, example DAO and demo test contracts
const migrationResult = await DAOstackMigration.migrate(options); // migrate
```

Reply:
> Why 'private' ?
> `DAOstackMigration.migration('private');`

Private means local testnet e.g. Ganache

Reply:
@orenyodfat `const migrationDAOResult = await DAOstackMigration.migrateDAO(options);` this is how you create the DAO with JS, also the `options` variable is detailed in that script. I can't really think of a good way to simplify that, if i write a new script it will be almost the same as the first just minus a few lines.

Reply:
> > Why 'private' ?
> > `DAOstackMigration.migration('private');`
> 
> Private means local testnet e.g. Ganache

Yea.. need to document that .
How about adding a var network = 'private' and explain that it can be any supported network.

Reply:
> We need a simple instruction to setup a custom dao . not the whole migration.
> as below...
> 
> ```
> // migrate base contracts
> const migrationBaseResult = await DAOstackMigration.migrateBase(options);
> migrationBaseResult.base.GenesisProtocol // migrated genesis protocol address
> // migrate an example DAO (requires an existing `output` file with a base migration)
> const migrationDAOResult = await DAOstackMigration.migrateDAO(options);
> migrationDAOResult.dao.Avatar // DAO avatar address
> // migrate an demo test scenario (requires an existing `output` file with a base migration)
> const migrationDemoResult = await DAOstackMigration.migrateDemoTest(options);
> migrationDemoResult.test.Avatar // Test DAO avatar address
> // migrate base, example DAO and demo test contracts
> const migrationResult = await DAOstackMigration.migrate(options); // migrate
> ```

Is that been addressed ?